.github/workflows/go-template-ci.yml

```diff
@@ -40,15 +40,7 @@ jobs:
           fail_ci_if_error: true
           files: ./coverage.out
           verbose: true
-      - name: SonarCloud Scan
```

Comment:
please point to the hosting at sonar.wednesday.is You can take a look [here](https://github.com/wednesday-solutions/go-template/blob/master/.github/workflows/go-template-ci.yml#L48)